build-system/tasks/karma.conf.js

```diff
@@ -179,7 +177,10 @@ module.exports = {
     'karma-sauce-launcher',
     'karma-sinon-chai',
     {
-      'middleware:custom': ['factory', function() {return app;}],
+      'middleware:custom': ['factory', function() {
+        var app = require(require.resolve('../server.js'));
```

Comment:
yeah lets to that, move it to a start-server.js and in the origin server file just export the server